docs/diagrams/ModelClassDiagram.puml

```diff
@@ -5,12 +5,11 @@ skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
 Package Model <<Rectangle>>{
-Interface ReadOnlyAddressBook <<Interface>>
-Interface Model <<Interface>>
-Interface ObservableList <<Interface>>
+Class "<<Interface>>\nReadOnlyAddressBook"
+Class "<<Interface>>\nModel"
+Class "<<Interface>>\nObservableList"
+
```

Comment:
Interface class should be in non-italics
https://github.com/nus-cs2103-AY2021S2/forum/issues/276